bootstrapping-lambda/src/reporting/reportingServiceClient.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  GrafanaRequest,
+  StageMetric,
+  MetricsResponse,
+  Metric,
+} from "../../../shared/types/grafanaType";
+import * as AWS from "aws-sdk";
+import { standardAwsConfig } from "../../../shared/awsIntegration";
+
+const metricEndpointMap = {
+  [StageMetric.UNIQUE_USERS_CODE]: "CODE",
+  [StageMetric.UNIQUE_USERS_PROD]: "PROD",
+};
+
+const stageMetricToMetric = {
+  [StageMetric.UNIQUE_USERS_CODE]: Metric.UNIQUE_USERS,
+  [StageMetric.UNIQUE_USERS_PROD]: Metric.UNIQUE_USERS,
+};
+
+export const getMetrics = async (
+  request: GrafanaRequest
+): Promise<MetricsResponse[]> => {
+  const lambda = new AWS.Lambda(standardAwsConfig);
+  const { range, targets } = request;
+  const metricsResponse = await Promise.all(
+    //FIXME perhaps allSettled here and then log unsuccessful attempts?
+    targets.map((target) => {
+      console.log(`processing grafana request`, target.target);
+      return lambda
+        .invoke({
+          FunctionName: `pinboard-database-bridge-lambda-${
```

Comment:
this could do with being refactored into a function (which takes stage as a param) in shared, and referenced in `stack.ts`, just like we do for https://github.com/guardian/pinboard/blob/6ef9bf05b0dfe7661442e67e7ef705055b87cbad/shared/constants.ts#L11